SIT.Manager/Models/Tools/PortCheckerResponse.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace SIT.Manager.Models.Tools;
+
+public class PortCheckerResponse
+{
+    [JsonPropertyName("akiSuccess")] public bool AkiSuccess { get; init; } = false;
+    [JsonPropertyName("natSuccess")] public bool NatSuccess { get; init; } = false;
+    [JsonPropertyName("relaySuccess")] public bool RelaySuccess { get; init; } = false;
+
+    [JsonPropertyName("portsUsed")] public PortCheckerPorts PortsUsed { get; init; } = new();
+
+    [JsonPropertyName("ipAddress")] public string? IpAddress { get; init; } = "x.x.x.x";
+}
+
+public class PortCheckerPorts
+{
+    [JsonPropertyName("akiPort")] public string AkiPort { get; set; } = "6969";
+    [JsonPropertyName("relayPort")] public string RelayPort { get; set; } = "6970";
+    [JsonPropertyName("natPort")] public string NatPort { get; set; } = "6971";
```

Comment:
Am I being stupid or is there not 4 ports, 6972 as well right? Or does that one not really matter?

Reply:
6972 is used but it's a little weird. I don't know the exacts around it but it's just for stuff like insurance and flea market messages and it always seems to show up closed on port tests. There might be a way to test it but that's beyond my knowledge so we'd need to ask bullet or one of the others